_po/ja/how-to/laravel.po

```diff
@@ -288,6 +415,16 @@ msgid ""
 "        });\n"
 "    }"
 msgstr ""
+"        Schema::table('posts', function (Blueprint $table) {\n"
+"            DB::statement('CREATE EXTENSION IF NOT EXISTS pgroonga');\n"
+"            DB::statement('CREATE INDEX pgrn_posts_id_index ON posts USING pgr"
+"oonga (id)');\n"
+"            DB::statement('CREATE INDEX pgrn_posts_title_index ON posts USING "
+"pgroonga (title)');\n"
+"            DB::statement('CREATE INDEX pgrn_posts_body_index ON posts USING p"
+"groonga (body)');\n"
+"        });\n"
+"    }"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.